oss_local_scripts/make_egg.sh

```diff
@@ -103,6 +103,9 @@ build_source() {
   echo -e "\n\n\n================= Build ${BUILD_NUMBER} ================\n\n\n"
   # Configure
   cd ${WORKSPACE}
+  # Clean up, except for the deps.
+  rm -rf "${WORKSPACE}/debug" "${WORKSPACE}/release"
```

Comment:
True, although the travis logs show ccache is activated, so it shouldn't be too bad there.